SECURITY.md

```diff
@@ -0,0 +1,76 @@
+# Security and OpenColorIO
+
+The OpenColorIO Technical Steering Committee (TSC) takes security very
+seriously. We strive to design secure software, and utilise continuous 
+integration and code analysis tools to help identify potential 
+vulnerabilities.
+
+Users should exercise caution when working with untrusted data (config files, 
+LUTs, etc.). OCIO takes every precaution to read only valid data, but it 
+would be naive to say our code is immune to every exploit.
+
+## Reporting Vulnerabilities
+
+Quickly resolving security related issues is a priority. If you think
+you've found a potential vulnerability in OpenColorIO, please report it by
+emailing security@opencolorio.org. Only TSC members and ASWF project
+management have access to these messages.
+
+Include detailed steps to reproduce the issue, and any other information that
+could aid an investigation. Someone will assess the report and make every
+effort to respond within 14 days.
+
+## Outstanding Security Issues
+
+None
+
+## Addressed Security Issues
+
+None
+
+## File Format Expectations
+
+Attempting to read an OCIO config (YAML) file will:
+* Return success and produce a valid Config data structure in memory
+* Fail with an error
+
+The vast majority of data used by OCIO comes from user-supplied transform 
+files referenced in a config. These files are arbitrarily large, may live 
+on any accessible volume, and are loaded lazily when requested by a Processor. 
+Referenced file paths may contain environment variables for expansion at load 
+time. Config authors should consider the implications of this when defining 
+file and search paths, to avoid the possibility of a maliciously modified 
+environment redirecting file reads to an insecure location.
+
+It is a bug if some file causes the library to crash. It is a serious
+security issue if some file causes arbitrary code execution.
+
+OpenColorIO will attempt to associate a file's data and layout with a 
+registered file format, regardless of a file's extension. Only expected data 
+structures will be read. It is a bug if reading invalid or malformed data from 
+a file does not result in an immediate error.
```

Comment:
OCIO tries all the format readers until one succeeds.  No error is raised if a format fails and then another succeeds.  Also, we purposely allow some malformed data, e.g. common misspellings such as SatNode/SATNode.  So I think the last sentence should be modified.  

Suggest: "It is a bug if reading seriously invalid or malformed data from a file does not result in an error from the file format reader ultimately used with a file."